content/blog/day-2-kubernetes-migrating-eks-nodegroups-with-zero-downtime/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Day 2 Kubernetes - Migrating EKS Node Groups with Zero Downtime"
+authors: ["mike-metral"]
+tags: ["Kubernetes","EKS", "Migrations", "Node Groups"]
+date: "2019-07-22"
+
+meta_image: "eks-migrate-nodegroups.png"
+meta_description: "Using Pulumi, we'll walk through Day 2 Kubernetes cluster administration by migrating a workload across EKS node groups with zero downtime." 
```

Comment:
This field is actually optional (and unused in the new blog), but if you do want to specify an explicit `<meta name="description" content="[...]">`, the property should be named `meta_desc`, not `meta_description`. When omitted, we just use the `.Summary` property supplied by Hugo (which is derived from the `<!--more-->` comment you've provided in your post). In general, if you've written an intro paragraph suitable for a home-page and/or social-media preview, you needn't specify anything for this.

Reply:
Thanks for the tip! I've removed `meta_description` in favor of the `.Summary` which is better detailed.